### PR TITLE
{chem}[foss/2021a] pyWannier90 v2021-12-07 w/ Python 3.9.5

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3116.skitty.os - Linux RHEL 8.4, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/0ded7a85dac99f05160ff7322ecd1108 for a full test report.

File: easybuild/easyconfigs/p/pyWannier90/pyWannier90-2021-12-07-foss-2021a.eb [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+builddependencies = [
+    ('pybind11', '2.6.2'),
+]
+
+local_pymaj = '3'
+local_pymin = '9'
+local_pymajmin = local_pymaj + local_pymin
+
+dependencies = [
+    ('Python', '%s.%s.5' % (local_pymaj, local_pymin)),
```
Comment (jfgrimm) [inline easybuild/easyconfigs/p/pyWannier90/pyWannier90-2021-12-07-foss-2021a.eb:40]: what's the point of this? I don't see `local_pymajmin` used anywhere

**Review by jfgrimm — APPROVED** [review]
LGTM

**Comment by jfgrimm** [comment]
Going in, thanks @boegel!

**Pull request merged**